v-next/hardhat-node-test-reporter/integration-tests/fixture-tests/asynchronous-activity-test/result.txt

```diff
@@ -0,0 +1,14 @@
+
+  [90m[32m✔[39m[90m foo[39m
+
+  [31m1) integration-tests/fixture-tests/asynchronous-activity-test/test.ts[39m [31m[3m(203ms)[23m[39m
+
+
+[32m1 passing[39m[90m (234ms)[39m[31m[39m
+[31m1 failing[39m
+[34mℹ[39m Error: Test "foo" at integration-tests/fixture-tests/asynchronous-activity-test/test.ts:1:29 generated asynchronous activity after the test ended. This activity created the error "Error" and would have caused the test to fail, but instead triggered an uncaughtException event.
```

Comment:
TODO(@galargh): We need to handle the path separators here correctly so that the integration tests pass on Windows.